src/Explorer/Controls/NotebookGallery/CodeOfConductComponent.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { shallow } from "enzyme";
+import React from "react";
+import { CodeOfConductComponent, CodeOfConductComponentProps } from "./CodeOfConductComponent";
+
+describe("CodeOfConductComponent", () => {
+  it("renders", () => {
+    const props: CodeOfConductComponentProps = {
+      junoClient: undefined,
+      onAcceptCodeOfConduct: undefined
+    };
+
+    const wrapper = shallow(<CodeOfConductComponent {...props} />);
+    expect(wrapper).toMatchSnapshot();
+  });
```

Comment:
It would be good if you can simulate a click on the submit button and verify that it makes a remote call with checkbox on.